frontend/src/pages/learn/challenge-solution/[moduleId].tsx

```
@@ -0,0 +1,82 @@
+                    fontSize: 14,
+                    minimap: {
+                        enabled: false,
+                    },
+                    readOnly: true,
+                    contextmenu: false,
+                }}
+            />
+            <button
+                onClick={handleComplete}
+                aria-label="Mark code solution as complete"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)